hatchery/hatchery.go

```diff
@@ -673,8 +689,11 @@ func mountFiles(w http.ResponseWriter, r *http.Request) {
 		WorkspaceFlavor: "nextflow",
 	})
 	// Look for any `license` configs in containers
+	// TODO: decide if we want to remove this block now that we do not read stata here.
 	for _, v := range Config.ContainersMap {
-		if v.License.Enabled {
+		configFilePath := strings.ToLower(v.License.FilePath)
+		if v.License.Enabled && !strings.Contains(configFilePath, "stata") {
```

Comment:
why do we need this additional condition of `!strings.Contains(configFilePath, "stata")` here? Is it for the case for none stata licenses we are still going to mount them as files?

Reply:
Well, the intent was to skip only license paths with `stata` in them but continue to mount any others. It might be cleaner to remove the `for` loop since we don't have any other licensePaths set up for hatchery config. Should we just remove the `for` loop?